### ISSUE TITLE
Always issues "Multiple projects found, consider using a single `tsconfig` with `references` to speed up, or use `noWarnOnMultipleProjects` to suppress this warning"

### ISSUE THREAD

**Issue by Rumperuu** [import-js/eslint-import-resolver-typescript#484]
I have set up my codebase to use TypeScript's Project References, but I am still getting this warning every time I run ESLint with this plugin active.

My codebase is a monorepo. I am using Solution-Style Project References (with a single `tsconfig.json` at the project root with with an empty array for `files` and `references` to all of the individual projects, which each have their own `tsconfig.json`s). https://github.com/import-js/eslint-import-resolver-typescript/issues/474 suggests that this plugin cannot follow Solution-Style Project References without them being explicitly defined in a `project` array, but I've done this and I'm still getting the warning.

Relevant section of my `eslint.config.js` file:

```js
⋮
{
    settings: {
        'import-x/resolver-next': [
            createTypeScriptImportResolver({
                alwaysTryTypes: true,
                project: ['tsconfig.json', 'tsconfig.test.json', 'packages/*/tsconfig.json']
            })
        ]
    }
},
⋮
```

Root `tsconfig.json`:

```json
{
    "files": [],
    "references": [
        { "path": "./packages/common" },
        { "path": "./packages/common-back" },
        { "path": "./packages/back" },
        { "path": "./packages/common-front" },
        { "path": "./packages/front" },
    ],
    "include": [],
    "exclude": ["node_modules/**/*", "*/**/.jest/**/*", "*/**/*.test.*"]
}
```

- `eslint` 9.38.0
- `typescript` 5.9.3
- `typescript-eslint` 8.46.2
- `eslint-plugin-import-x` 4.16.1
- `eslint-import-resolver-typescript` 4.4.4

**Comment by Rumperuu** [comment]
On closer inspection, the [warning is just triggered by `projects.length` being greater than 1](https://github.com/import-js/eslint-import-resolver-typescript/blob/724c47d8c08fa8ffa702c7a94ba53b367a4727fb/src/normalize-options.ts#L65-L73). So this is a different consequence of the same problem described in https://github.com/import-js/eslint-import-resolver-typescript/issues/474, which should be resolved in https://github.com/import-js/eslint-import-resolver-typescript/issues/470

I'll leave this issue open because the exact wording of the warning will be helpful for other people searching (it took me a while to even realise that it was this package throwing the warning, rather than `eslint` or `typescript-eslint`)
